docs/how-do-i-install-my-operator-with-olm.md

```diff
@@ -28,23 +42,146 @@ spec:
+  UID:                 2864cc19-0ad4-11ea-8c0e-aa8cc6bce8d3
+Spec:
+  Channel:           stable
+  Name:              my-operator
+  Source:            my-catalog
+  Source Namespace:  olm
+Status:
+  Catalog Health:
+    Catalog Source Ref:
+      API Version:       operators.coreos.com/v1alpha1
+      Kind:              CatalogSource
+      Name:              my-catalog
+      Namespace:         olm
+      Resource Version:  787
+      UID:               674c52e3-0ad2-11ea-8c0e-aa8cc6bce8d3
+    Healthy:             true
+    Last Updated:        2019-11-19T13:54:51Z
+  Conditions:
+    Last Transition Time:  2019-11-19T13:54:51Z
+    Message:               all available catalogsources are healthy
+    Reason:                AllCatalogSourcesHealthy
+    Status:                False
+    Type:                  CatalogSourcesUnhealthy
+  Current CSV:             my-operator.v1.15.0
+  Install Plan Ref:
+    API Version:       operators.coreos.com/v1alpha1
+    Kind:              InstallPlan
+    Name:              install-4xhbl
+    Namespace:         operators
+    Resource Version:  1744
+    UID:               286e5546-0ad4-11ea-8c0e-aa8cc6bce8d3
+  Installed CSV:       my-operator.v1.15.0
+  Installplan:
+    API Version:  operators.coreos.com/v1alpha1
+    Kind:         InstallPlan
+    Name:         install-4xhbl
+    Uuid:         286e5546-0ad4-11ea-8c0e-aa8cc6bce8d3
+  Last Updated:   2019-11-19T13:54:55Z
+  State:          AtLatestKnown
+Events:           <none>
 ```
-$ kubectl get csv -n <namespace-operator-was-installed-in>
+
+The `Subscription` has successfully deployed the Operator when the `status.state` property reaches `AtLatestKnown`.
+
+You can verify that the Operator is deployed by checking the `ClusterServiceVersion` (CSV) object in the same namespace where the `Subscription` was placed in. The name of the CSV is referenced in `status.installedCSV` of the `Subscription`:
+
+```bash
+$ kubectl get csv my-operator.v1.15.0 -n olm
 
 NAME                  DISPLAY          VERSION           REPLACES              PHASE
-<name-of-csv>     <operator-name>     <version>  <csv-of-previous-version>   Succeeded
+my-operator.v1.15.0   My Operator      1.15.0            my-operator.v1.14.0   Succeeded
 ```
+
+If the the CSV has transitioned to the _Succeeded_ phase the Operator deployment was successful. You inspect it manually like so:
+
+```bash
+$ kubectl get deployments -n olm
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+my-operator             1/1     1            1           9m48s
 ```
-$ kubectl get deployments -n <namespace-operator-was-installed-in>
-NAME                         READY   UP-TO-DATE   AVAILABLE   AGE
-<name-of-your-operator>      1/1     1            1           9m48s
+
+If the ClusterServiceVersion fails to show up or does not reach the `Succeeded` phase, please check the [troubleshooting documentation](troubleshooting.md) to debug your installation.
```

Comment:
Same comment as above about the (troubleshooting.md#clusterserviceversion-troubleshooting).